_docs/releasing_gem.md

```diff
@@ -48,18 +49,43 @@ Get community feedback on the release pr, label it with skip-changelog, get it m
 
 ## Part 2: Actually create the release
 
-Checkout an updated copy of master
+Checkout an updated copy of voxpupuli master. Do not use a fork here. Or add voxpupuli as a additional remote to your fork.
+
+with the main repo
+
+```bash
+git switch master; git pull origin master
+```
+
+with a fork
 
 ```bash
-git checkout master; git pull origin master
+git remote add voxpupuli git@github.com:voxpupuli/$my_repo.git
+git fetch voxpupuli --tags
+git switch master
+git pull -r voxpupuli master
```

Comment:
```suggestion
git pull --rebase voxpupuli master
```

Not sure if the `--rebase` is really needed for this doc.  Anyway, using long-options is usually preferable for documentation purpose I think.